content/docs/command-reference/pull.md

```diff
@@ -55,6 +55,9 @@ dependencies backward from the target [stage files](/doc/command-reference/run),
 through the corresponding [pipelines](/doc/command-reference/pipeline), to find
 data files to pull.
 
+If `targets` are files/directories tracked by DVC, DVC will download cache for
+them, skipping the rest.
```

Comment:
```suggestion
If `targets` are files/directories tracked by DVC, DVC will download
them, skipping the rest.
```

Reply:
This behavior is probably already explained somewhere else in the doc (not sure).

And, how is this related to granularity?

Reply:
OK I changed it to "only download those targets", which starts to get to the granularity question, but needs more explanation, probably?

Reply:
Rewritten in 489a6c11